hack/update-generated-crd-code.sh

```diff
@@ -15,13 +15,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-HACK_DIR=$(dirname "${BASH_SOURCE}")
-REPO_ROOT=${HACK_DIR}/..
-
-${REPO_ROOT}/vendor/k8s.io/code-generator/generate-groups.sh \
+VERSION=$(go list -m all | grep k8s.io/code-generator | rev | cut -d"-" -f1 | cut -d" " -f1 | rev)
+TMP_DIR=$(mktemp -d)
+git clone https://github.com/kubernetes/code-generator.git ${TMP_DIR}
+(cd ${TMP_DIR} && git reset --hard ${VERSION})
+${TMP_DIR}/generate-groups.sh \
```

Comment:
Please note, if `contour` is checked out outside of $GOPATH, `code-generator` would still generate the CRD code into $GOPATH/src/github.com/heptio/contour/apis anyway due to https://github.com/kubernetes/code-generator/issues/57

Reply:
Yep, but i think this is assumed as per the [contributing guidelines](https://github.com/heptio/contour/blob/master/CONTRIBUTING.md).

Reply:
Cool, then we should be good